### PR TITLE
[zh-CN] update 'column-rule-width'

### PR DESCRIPTION
**Pull request by T34-active**
### Description

update content in [zh-CN](https://developer.mozilla.org/en-US/docs/Web/CSS/column-rule-width)

### Motivation

[zh-CN](https://developer.mozilla.org/zh-CN/docs/Web/CSS/column-rule-width) content does not match [en-US](https://developer.mozilla.org/en-US/docs/Web/CSS/column-rule-width) content

### Related issues and pull requests
[#18033](https://github.com/mdn/translated-content/pull/18033)
[2a9498b](https://github.com/mdn/translated-content/commit/2a9498b463ca831dab28c26a79b43f78bc640ac6)

### CODE REVIEW

File: files/zh-cn/web/css/column-rule-width/index.md [hunk 1]
```diff
@@ -5,46 +5,82 @@ slug: Web/CSS/column-rule-width
```
Comment (JasonLamv-t) [inline files/zh-cn/web/css/column-rule-width/index.md:5]: Please fill in the missing metadata about l10n.

File: files/zh-cn/web/css/column-rule-width/index.md [hunk 2]
```diff
@@ -5,46 +5,82 @@ slug: Web/CSS/column-rule-width
...
+column-rule-width: initial;
+column-rule-width: revert;
+column-rule-width: revert-layer;
+column-rule-width: unset;
 ```
 
-### Values
+`column-rule-width` 属性被指定为单个 `<'border-width'>` 值。
+
+### 取值
```
Comment (JasonLamv-t) [inline files/zh-cn/web/css/column-rule-width/index.md:34]:
```suggestion
### 值
```

File: files/zh-cn/web/css/column-rule-width/index.md [hunk 3]
```diff
@@ -5,46 +5,82 @@ slug: Web/CSS/column-rule-width
...
+
+## 示例
+
+### 设置属性宽度为 4px 的列规则
+
+#### HTML
+
+```html
+<p>
+  这是分成三列的文本。在<code>column-rule-width</code>属性中用于更改列与列之间的线条宽度。修改列之间线条的宽度。你不觉得这很棒吗？
```
Comment (JasonLamv-t) [inline files/zh-cn/web/css/column-rule-width/index.md:55]: 修改为 code tag 似乎不会有额外的渲染效果优化，这里与原文保持一致
```suggestion
  这是分成三列的文本。在 `column-rule-width` 属性中用于更改列与列之间的线条宽度。修改列之间线条的宽度。你不觉得这很棒吗？
```

**Review by JasonLamv-t — COMMENTED** [review]
一些建议

File: files/zh-cn/web/css/column-rule-width/index.md (hunk 3 above)
Comment (jasonren0403) [reply files/zh-cn/web/css/column-rule-width/index.md:55]: 还是用 code tag 好些，毕竟上下文是一段 HTML 代码

image

**Review by JasonLamv-t — APPROVED** [review]
Thank you! @T34-active

**Pull request merged**